src/client/search/SearchTermsReducer.js

```diff
@@ -0,0 +1,27 @@
+const SearchTermsReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'ADD_SEARCH_TERM':
+      return [
+      ...state,
+      SearchTermReducer(undefined, action),
+			];
+    default:
```

Comment:
Curly braces even for single line statements please, easy to add a debugging "console.log" or whatever and mess this up